### DISCUSSION TITLE
The image is not showing when I am working on it locally with a production database (remotely accessed database).

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10142] [Q&A]
The image is not showing when I am working on it locally with a production database (remotely accessed database).

**Comment by devansh-webkul** [comment]
Hello there,

Please check whether these extensions are installed or not,
https://devdocs.bagisto.com/2.2/introduction/requirements.html#php-extensions

We are using intervention package for image resizing and also check storage linking for other images to.

**Reply by sudeep2003** [reply]
PHP-GD is enabled, and the images work perfectly when using the local database. However, when accessing the database remotely, the images do not display. I've already verified that the hostname, database name, and password in the .env file are correct.

**Comment by sudeep2003** [comment]
PHP-GD is enabled, and the images work perfectly when using the local database. However, when accessing the database remotely, the images do not display. I've already verified that the hostname, database name, and password in the .env file are correct.

**Reply by suraj-webkul** [reply]
Hello, [sudeep2003](https://github.com/sudeep2003),

PHP-GD is enabled, but the images are still not displaying. This could be due to a broken link. Could you please provide a screenshot of the <img> tag from the inspect element and also share the main URL of your application?
